app/models/user.rb

```diff
@@ -16,4 +22,12 @@ def normalize_queue_item_positions
   def queued?(video)
     queue_items.map(&:video).include? video
```

Comment:
Just so you know, a more performant way to do this would be:

```ruby
queue_items.pluck(:video_id).include? video.id
```

Pluck pulls only the ids from the database and loads the data into an array. It saves a lot of processing time and power over loading every video into memory using `map`.